tls/s2n_security_policies.c

```diff
@@ -682,6 +682,14 @@ const struct s2n_security_policy security_policy_20210816_gcm = {
     .ecc_preferences = &s2n_ecc_preferences_20210816,
 };
 
+const struct s2n_security_policy security_policy_20220805 = {
+    .minimum_protocol_version = S2N_TLS13,
```

Comment:
I believe the minimum should be TLS1.2

Reply:
With this error in mind, can you test that this security policy can be successfully used to establish a connection, probably with both RSA and EC certs? A manual test using s2nc and s2nd should be sufficient.

Reply:
Updated description to add the manual testing and results